src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,201 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class TextBox
+		{
+			public static class Resources
+			{
+				public static class DeleteButton
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForeground")]
+						public static ResourceValue<Brush> Default => new("TextBoxDeleteButtonForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("TextBoxDeleteButtonForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("TextBoxDeleteButtonForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "TextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("TextBoxDeleteButtonForegroundDisabled", true);
+					}
+				}
+
+				public static class Filled
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForeground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForeground")]
+						public static ResourceValue<Brush> Placeholder => new("FilledTextBoxPlaceholderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PlaceholderPointerOver => new("FilledTextBoxPlaceholderForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundFocused")]
+						public static ResourceValue<Brush> PlaceholderFocused => new("FilledTextBoxPlaceholderForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxPlaceholderForegroundDisabled")]
+						public static ResourceValue<Brush> PlaceholderDisabled => new("FilledTextBoxPlaceholderForegroundDisabled", true);
```

Comment:
These should be moved to a `PlaceholderForeground` class within the `Filled` class